packages/react-redux-subspace/.babelrc.js

```diff
@@ -0,0 +1,14 @@
+const { BABEL_ENV, NODE_ENV } = process.env
+
+const cjs = BABEL_ENV === 'cjs' || NODE_ENV === 'test'
+
+module.exports = {
+  presets: [
+  	['env', { loose: true, modules: false }],
+  	'react',
+  	'stage-3',
+  ],
+  plugins: [
+  	cjs && 'transform-es2015-modules-commonjs',
```

Comment:
Should this be [`loose` too?](https://github.com/gaearon/redux-thunk/blob/master/.babelrc#L26)

Reply:
I've recently started to omit `loose` of cjs transform because I feel the gain is minimal. The only thing it does is that it makes "magic" `__esModule` property enumerable (it's simply assigned and not defined with `defineProperty`) and that makes spreading namespaces weird, as in
```js
import * as namespace from './namespace'
const obj = { ...namespace } // oops, obj has `__esModule` property
```